modules/json-schema/src/internals/Extractors.scala

```diff
@@ -328,6 +333,43 @@ private[json_schema] object Extractors {
     }
   }
 
+  object CaseConst {
+    def unapply(sch: Schema): Option[(List[Hint], Primitive)] = {
+      sch match {
+        case const: ConstSchema =>
+          val genericHints = getGenericHints(sch)
+          val value = const.getPermittedValue()
+          val constHint = Hint.Const(toNode(value))
+          Some((constHint :: genericHints) -> Primitive.PFreeForm)
+        case _ => None
+      }
+    }
+
+    private def toNode(in: Object): Node = {
```

Comment:
We have another function like this one, although it does handle more types. I'm not sure if we want to use that same function here, or if we need to handle any of those same types here, but thought it was worth mentioning at least https://github.com/disneystreaming/smithy-translate/blob/2d388fe400b99e96538c7255e15df2221b8e81e6/modules/compiler-core/src/internals/GetExtensions.scala#L53

Reply:
The one difference is the handling of `org.json.JSONObject.NULL` which is the value that we get when a const has a `null` value.

`anyToNode` would work in its current implementation because that `NULL` value would go through the default case but I would rather have it explicitly handled.

However, adding a case for it in `anyToNode` would require adding a dependency on `org.json` in the compiler-core module. This is a transitive dependency of `everit-json-schema`, which is only used by the json-schema module.

So to keep the dependency to a minimum, I think I'd rather keep the local toNode implementation.